ui-tests/src/main/java/org/jenkinsci/test/acceptance/plugins/warnings_ng/DetailsTableRow.java

```diff
@@ -0,0 +1,33 @@
+package org.jenkinsci.test.acceptance.plugins.warnings_ng;
+
+import org.openqa.selenium.WebElement;
+
+/**
+ * Representation for the details row which can be toggled by clicking the icon in the details column on a issues-table
+ * row.
+ *
+ * @author Stephan Plöderl
+ */
+public class DetailsTableRow extends AbstractIssuesTableRow {
+    private final String details;
+
+    /**
+     * Creates a new representation for a issues-table details row.
+     *
+     * @param row
+     *         the WebElement representing the row.
+     */
+    DetailsTableRow(final WebElement row) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [It is a good practice to call super() in a constructor](https://app.codacy.com/manual/uhafner/warnings-ng-plugin/pullRequest?prid=5070362)